public/pages/MonitorDetails/containers/Triggers/Triggers.js

```diff
@@ -76,28 +76,29 @@ export default class Triggers extends Component {
 
   componentDidMount() {
     this.updateMonitorState();
+
+    if (this.props.delegateMonitors.length > 0) {
+      this.formatTriggerCondtions();
+    }
   }
 
   componentDidUpdate(prevProps) {
     if (this.props.monitor !== prevProps.monitor) {
       this.updateMonitorState();
     }
+
+    if (
+      this.props.delegateMonitors !== prevProps.delegateMonitors &&
+      this.props.delegateMonitors.length > 0
```

Comment:
Nitpick: Since the `this.props.delegateMonitors.length > 0` check is performed here and in `componentDidMount`, should this check just be moved to the `formatTriggerCondtions` method?